boost-gradle/src/main/groovy/io/openliberty/boost/gradle/tasks/BoostPackageTask.groovy

```diff
@@ -92,6 +100,16 @@ public class BoostPackageTask extends AbstractBoostTask {
                             project.bootRepackage.finalizedBy 'boostPackage'
                         }
                     }
+                } else if (isJ2EEProject()) {
```

Comment:
At this point you already know it's not a SB project. Does do `else` and remove `isJ2EEProject()` method